pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -3096,5 +3096,30 @@ public void removeSubscribeRate(@Suspended final AsyncResponse asyncResponse,
         });
     }
 
+    @DELETE
+    @Path("/{tenant}/{namespace}/{topic}/truncate")
+    @ApiOperation(value = "Truncate a topic.",
+            notes = "The latest ledger cannot be deleted ,and only delete acknowledged ledgers.")
```

Comment:
The latest ledger will not be deleted, and also only delete acknowledged ledgers can be deleted.

Reply:
Thank you very much, I will fix it as soon as possible.